plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -294,7 +294,7 @@ class FeatureDevController(
         val prevInsertAction = insertAction()
 
         if (action == "accept-change") {
-            session.insertChanges(
+            session.insertChangesWithoutUpdateFileComponents(
```

Comment:
I think the method name including "WithoutUpdateFileComponents" is an artifact of today's state of code and will not be clear to those reading a future state.

Can we refine this to be more focused on what is actually happening here? Something like `writeChangesToDisk`?